pkg/virt-operator/resource/generate/components/prometheus.go

```diff
@@ -573,3 +579,16 @@ func NewPrometheusRuleSpec(ns string, workloadUpdatesEnabled bool) *v1.Prometheu
 
 	return ruleSpec
 }
+
+func getRunbookURLTemplate() string {
+	runbookURLTemplate, exists := os.LookupEnv(runbookURLTemplateEnv)
+	if !exists {
+		runbookURLTemplate = defaultRunbookURLTemplate
```

Comment:
@enp0s3 I kept this check, although we set a default value in `config-default.sh`, let me know if that's ok.